src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/HearingGetConsumerTest.java

```diff
@@ -210,7 +195,7 @@ void shouldFailGetHearingWithBadRequest() {
                 SERVICE_AUTHORIZATION_TOKEN,
                 BAD_REQUEST_CASE_ID,
                 null
-            )).extracting("status").isEqualTo(BAD_REQUEST);
+            )).extracting(STATUS).isEqualTo(BAD_REQUEST);
```

Comment:
"status" can be replaced by STATUS constant.

Reply:
Yes, this is the change I made